.github/workflows/lint.yml

```
@@ -0,0 +1,13 @@
+name: Lint
+
+on: [push, pull_request]
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v2
+      - run: pip install isort
+      - name: Check formatting
+        run: make check
```

```suggestion
      - name: Check import order
        run: isort --profile black --check .
```

This is pretty nitpicky and 100% my personal opinion, but I'm not a huge fan of makefiles for non-compiled languages, esp. since `make` is pretty foreign to the Python ecosystem.

For now, I've suggested putting the `isort` invocation into the workflow config directly. If we ever end up needing extra formatting checks, I'll create a shell script to handle those.

---

Done.

---

Done!

As noted below, an alternative to a shell script could be to add a [setup.py command](https://docs.python.org/3/distutils/extending.html#integrating-new-commands) (so formatting would be done by `python setup.py format` or similar), let me know if you want me to do that.